packages/perseus/src/widgets/__tests__/interactive-graph.test.ts

```diff
@@ -170,7 +170,7 @@ describe("segment graph", () => {
         );
 
         // Act
-        await userEvent.type(movablePoints[0], "{arrowup}{arrowdown}");
+        await userEvent.type(movablePoints[1], "{arrowup}{arrowdown}");
```

Comment:
`movablePoints[0]` is now the center point of the segment, i.e. the one that moves the entire segment when dragged. It has to come first so the tab order is right (or maybe there's a way we can set the tabIndex?)